stdlib/stdLabels.mli

```diff
@@ -28,8 +28,6 @@
      let everything = Array.create_matrix ~dimx:42 ~dimy:42 42
    ]}
 
-   Detailed interfaces documenting the function argument labels can be found
-   at {!ArrayLabels}, {!BytesLabels}, {!ListLabels} and {!StringLabels}.
```

Comment:
Review note: the reason why this is unnecessary (was not immediately obvious to me and) is that the modules are mentioned right below, so the users can just click on them to get their documentation.